src/constellation/constellation.py

```diff
@@ -335,18 +335,39 @@ def remove(self):
         docker_util.remove_network(self.name)
 
 
-class ConstellationMount:
-    def __init__(self, name, path, **kwargs):
-        self.name = name
-        self.path = path
+# Base class for mounts.
+class _ConstellationMount:
+    def __init__(self, target, **kwargs):
+        self.target = target
         self.kwargs = kwargs
 
+    def to_mount(self, volumes):
+        """Convert to a Docker mount. Subclasses can use `volumes` if needed."""
+        raise NotImplementedError("Subclasses must implement `to_mount`")
```

Comment:
https://github.com/reside-ic/constellation/blob/master/pyproject.toml#L100

You can add `raise NotImplementedError` to that list to get coverage to shut up about these lines.

Reply:
https://coverage.readthedocs.io/en/latest/excluding.html